src/pages/ProductDetail/components/ProductImage/ProductImage.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import './ProductImage.scss';
+
+const IMAGE_WIDTH = 550;
+const MIN_GAP = IMAGE_WIDTH * 0.1; // 1/10 만큼 이동하면 다음 이미지로
+
+class ProductImage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      focusIdx: 0,
+    };
+    // 드래그 관련 Ref
+    this.isDragRef = React.createRef();
+    this.isDragRef.current = false;
```

Comment:
```suggestion
    this.isDragRef = React.createRef(false);
```
- isDrag 보다 좀 더 구체적으로 써주세요. 드래그가 가능하다는 건지, 드래그 중이라는 건지 이름만 봐서는 조금 헷갈립니다.
- 부여하고 싶은 초기값은 그냥 인자로 넣어주시면 될거 같아요

Reply:
- 처음에는 isPress로 했다가 드래그 관련인데 타당하지 않다고 생각해서 isDrag로 바꿨었습니다.. 혹시 isDrag 대신에 어떤 이름으로 해야 할까요?
-` React.createRef(false)`로 초기화 할 수 있다면 아래 Ref도 같은 방식으로 초기화 될 줄 알았는데 안되더라구요 혹시 어떤 경우에만 초기화가 가능한지 알려주실수 있으신가요?

```js
// 변경 전
this.dragPosRef = React.createRef();
this.dragPosRef.current = {
      originPosX: 0,
      startPosX: 0,
};

//  변경 후
this.dragPosRef = React.createRef({
   originPosX: 0,
   startPosX: 0,
});
```